coord/src/info_command.c

```diff
@@ -254,6 +264,13 @@ static void generateFieldsReply(InfoFields *fields, RedisModuleCtx *ctx) {
   RedisModule_ReplySetArrayLength(ctx, nCursorStats);
   n += 2;
 
+  RedisModule_ReplyWithSimpleString(ctx, "dialect_stats");
+  RedisModule_ReplyWithArray(ctx, REDISMODULE_POSTPONED_ARRAY_LEN);
+  size_t nDialectStats =
+      replyKvArray(fields, ctx, fields->dialectValues, dialectSpecs, NUM_DIALECT_FIELDS_SPEC);
+  RedisModule_ReplySetArrayLength(ctx, nDialectStats);
+  n += 2;
```

Comment:
We can probably have a macro for all of the above. Maybe have another PR for this.

Reply:
Will do in a subsequent PR.